### DISCUSSION TITLE
What is error 500?

### DISCUSSION THREAD

**Discussion by Dev-Rocking** [community/community#66043] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

What could be the cause for this?

**Comment by ghostinhershell** [comment]
Hey @Dev-Rocking, 

A "500" is an internal server error, which means an error occurred in the GitHub application code. It usually happens when GitHub server encounters a problem and is down. You can check the [Github Status Website](https://githubstatus.com/) to keep track of the status of our servers, including whether or not a server is down.
